libraries/bezier-rs/src/bezier/lookup.rs

```diff
@@ -21,11 +23,43 @@ impl Bezier {
 		}
 	}
 
+	pub(crate) fn unrestricted_euclidean_evaluate(&self, d: f64, error: f64) -> DVec2 {
```

Comment:
Add a description for the function. 

It's not clear to me whether d is the absolute distance, or the relative distance (i.e. relative to the total length). Maybe clarify this in your description.